test-app/package.json

```diff
@@ -56,6 +57,7 @@
     "keycodes-enum": "0.0.2",
     "lodash": "^4.17.21",
     "react": "^16.6.3",
-    "react-dom": "^16.6.3"
+    "react-dom": "^16.6.3",
+    "treeverse": "^3.0.0"
```

Comment:
These shouldn't be needed here, test-app should be using `treeverse` through CAM (since it's a runtime dependency, not a peerDependency), and the test app doesn't use React :)

Reply:
@mmoss  test-app is not able to use treeverse through CAM. Might be an npm Issue!
![Screenshot 2022-11-29 at 10 00 50 PM](https://user-images.githubusercontent.com/66003209/204590306-1c415204-50f3-44c7-8238-8576f08fac32.png)

Reply:
@tussharrane looks like an issue with using the relative dependency (`file:..`) in the test-app `package.json` and Webpack's module resolution. I was able to get it working by removing [our customization of the module resolution here](https://github.com/CurriculumAssociates/createjs-accessibility/blob/32399bb512286939198ca42219c44f9ee4acf597/test-app/webpack.config.js#L71-L73). Removing that line and letting Webpack use its default module resolution seems to fix the issue. Can you give that a try?

Reply:
@mmoss yes, removing that line seems to be working locally but, `lint and build test-app` job is failing.